packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -473,6 +474,13 @@ export default class CodeMode extends Component<Signature> {
     }
   }
 
+  get isInstance() {
+    return (
+      isReady(this.openFile.current) &&
+      this.openFile.current.name.endsWith('.json')
+    );
```

Comment:
this isn't quite right--consider the `package.json` file--it's not a card instance right? --or maybe the user deletes the `meta` property from the instance JSON file. is have type assertion functions that you can use to assert that the contents of a json file are indeed a card instance.

Reply:
I see. Do you mean `isCardDocument`?